code/obj/decal.dm

```diff
@@ -366,6 +366,7 @@ obj/decal/fakeobjects/teleport_pad
 	icon = 'icons/obj/atmospherics/pipes/regular_pipe.dmi'
 	icon_state = "intact"
 	anchored = 1
+	layer = 2.9
```

Comment:
use defines please

Reply:
i've used `DECAL_LAYER` which is 2.2 (`TURF_LAYER+0.2)